README_OLD.md

```diff
@@ -0,0 +1,251 @@
+Authorize.Net PHP SDK
+======================
+
```

Comment:
I don't think we should have a README_OLD, rather a DeprecatedSamples.md perhaps.  It should not have the old readme but just the old usage examples with a small explanation